jekyll/_cci2/large-language-model-evaluation.adoc

```diff
@@ -0,0 +1,100 @@
+---
+contentTags:
+  platform:
+    - Cloud
+    - Server v4.x
+---
+= Large Language Model evaluation
```

Comment:
I am looking again at the titles of the sections next to this one. 
And I wonder if a title like this would be better:
> Testing LLM-enabled applications through evaluations